mne/viz/_3d.py

```diff
@@ -777,7 +786,95 @@ def _limits_to_control_points(clim, stc_data, colormap):
             bump = 1e-5 if ctrl_pts[0] == ctrl_pts[1] else -1e-5
             ctrl_pts[1] = ctrl_pts[0] + bump * (ctrl_pts[2] - ctrl_pts[0])
 
-    return ctrl_pts, colormap
+    # Construct cmap manually if 'mne' and get cmap bounds
+    # and triage transparent argument
+    if colormap in ('mne', 'mne_analyze'):
+        colormap = mne_analyze_colormap(ctrl_pts)
+        scale_pts = [-1 * ctrl_pts[-1], 0, ctrl_pts[-1]]
+        transparent = False if transparent is None else transparent
+    else:
+        scale_pts = ctrl_pts
+        transparent = True if transparent is None else transparent
+
+    return scale_pts, colormap, transparent
+
+
+def _scale_mayavi_lut(lut_manager, fmin, fmid, fmax, transparent,
+                      verbose=None):
+        """Scale a mayavi colormap LUT to a given fmin, fmid and fmax.
+
+        This function operates on a Mayavi LUTManager. This manager can be
+        obtained through the traits interface of mayavi. For example:
+        `x.module_manager.vector_lut_manager`.
```

Comment:
double-backticks since this is really code (won't link properly if it ever renders)